src/services/dumpers/agent-nodejs.ts

```diff
@@ -82,29 +83,66 @@ export default class AgentNodeJs extends AbstractDumper {
       }
     }
 
-    const scripts: { [name: string]: string } = {
+    let scripts: { [name: string]: string } = {
       start: 'nodemon ./index.js',
       'start:agent': 'node ./index.js',
     };
     const devDependencies: { [name: string]: string } = {
       nodemon: '^2.0.12',
     };
+    const nodemonConfig = {
+      ignore: ['./forestadmin-schema.json'],
+    };
+
+    if (language === languages.Typescript) {
+      scripts = {
+        build: 'tsc',
+        start: 'nodemon ./index.ts',
+        'start:agent': 'node ./dist/index.js',
```

Comment:
Didn't notice in the previous PR, but usually `yarn/npm start` is the production command.

Reply:
Seen with @romain-gilliotte, at the end of the onboarding, a client will do a `yarn start` naturally. So we felt it removes friction if the client doesn't need to build first. Before putting the ccode in production, every client will lookup at least once the scripts in the package.json anyway.
Plus we followed what is done [here](https://github.com/ForestAdmin/agent-nodejs/blob/main/packages/_example/package.json) 🤷